dbms/src/Flash/Coprocessor/CoprocessorReader.h

```diff
@@ -88,12 +88,12 @@ class CoprocessorReader
         , resp_iter(std::move(tasks), cluster, concurrency, &Poco::Logger::get("pingcap/coprocessor"))
         , collected(false)
         , concurrency_(concurrency)
-    {
-        resp_iter.open();
-    }
+    {}
 
     const DAGSchema & getOutputSchema() const { return schema; }
 
+    void open() { resp_iter.open(); }
```

Comment:
How about adding some comments for `open()` and its usage on `TiRemoteBlockInputStream::readPrefixImpl` , so that others will know when `open()` be called in its lifecycle.

Reply:
updated, PTAL